app/src/main/java/com/eveningoutpost/dexdrip/Home.java

```diff
@@ -3016,6 +3018,7 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 break;
             case R.id.action_open_watch_settings:
                 startService(new Intent(this, WatchUpdaterService.class).setAction(WatchUpdaterService.ACTION_OPEN_SETTINGS));
```

Comment:
Yes, I'm afraid I never tested this.  I think this was existing code when I joined.